packages/sanity/src/core/releases/tool/detail/documentTable/DocumentActions.tsx

```diff
@@ -22,11 +21,13 @@ export const DocumentActions = memo(
     const [discardStatus, setDiscardStatus] = useState<'idle' | 'discarding' | 'error'>('idle')
     const client = useClient(DEFAULT_STUDIO_CLIENT_OPTIONS)
     const toast = useToast()
-    const {t} = useTranslation(releasesLocaleNamespace)
+    const {t} = useTranslation()
 
     const handleDiscardVersion = async () => {
       try {
         setDiscardStatus('discarding')
+
+        // TODO: should we use the document operations for this?
```

Comment:
There was some discussion (either on slack or on linear) about how we aren't using document operations.
I considered moving this to somewhere that could be imported directly from both, but maybe since we are likely going to change this in the coming future we don't need to do much beyond the TODO?

Reply:
cc @bjoerge